src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForInitializerPadCheck.java

```diff
@@ -148,11 +148,9 @@ public int[] getRequiredTokens() {
     @Override
     public void visitToken(DetailAST ast) {
         if (!ast.hasChildren()) {
-            // empty for initializer. test pad before semi.
-            final DetailAST semi = ast.getNextSibling();
-            final int semiLineIdx = semi.getLineNo() - 1;
+            final int semiLineIdx = ast.getLineNo() - 1;
```

Comment:
This is no longer a semi, and needs to be renamed. Please make it `lineIdx`.

Reply:
done

Reply:
Not done, `lineIdx` should have a lowercase `l` not an uppercase one.